CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# How to contribute
+
+## Submitting changes
+
+Please send a [GitHub Pull Request to dosemu2](https://github.com/stsp/dosemu2/pull/new/master) with a clear list of what you've done (read more about [pull requests](http://help.github.com/pull-requests/)).  Please follow our coding conventions (below), make sure all of your commits are atomic (one feature per commit) and ensure at every commit in a series the code compiles.
+
+Always write a clear log message for your commits. One-line messages are fine for small changes, but bigger changes should look like this:
+
+    $ git commit -m "A brief summary of the commit
+    > 
+    > A paragraph describing what changed and its impact."
+
+## Coding conventions
+Since dosemu2 has a long history, there are many different coding styles in use. We'd like to standardise on a subtle variant of 
+the Kernighan & Ritchie style, but mass conversion is not really an option as we don't want to obscure the commit history. So
```

Comment:
This is a coding style, so please make
the lines shorter. :) They wrap.

Reply:
Yes, I'll wrap them manually, I didn't consider you might look at it as a plain text file. The Markdown format is a little like html it usually has no newlines within the paragraph, then it wraps when rendering.